pages/team/new/index.js

```diff
@@ -99,9 +99,9 @@ function CreateNewTeam({ router }) {
         <h2>Members</h2>
         <ul>
           {members.map((member, index) => (
-            <Member key={index}>
+            <MemberItem key={index}>
```

Comment:
❓ mh wait, shouldnt that be the `Member` component outside of styles.js now?

Reply:
Here we need a dumb styled component, not the one with remove member mutation functionally. Jan changed the styled component to "MemberItem" but didn't replace all instances where it's imported so this view was crashing

Reply:
ah ok, i thought you might want to also rm members on team creation

Reply:
Might be good, but that component is now coupled with the mutation and i didn't feel like refactoring. I'll do it if we have time later.